.github/workflows/osl_sync_main.yml

```diff
@@ -12,6 +12,7 @@ env:
 
 jobs:
   sync_main_apache:
+    if: ${{ vars.ENABLE_OSL_SYNC_MAIN == 'yes' }}
```

Comment:
Have u added this var to the Settings?

Reply:
I was waiting for your review to add it.
Now I added it.